Formula/overdrive.rb

```diff
@@ -11,10 +11,15 @@ class Overdrive < Formula
   end
 
   depends_on "tidy-html5"
+
   uses_from_macos "curl"
-  uses_from_macos "libressl"
   uses_from_macos "libxml2"
 
+  on_linux do
+    depends_on "openssl@1.1" # for openssl (non keg-only)
+    depends_on "util-linux" # for uuidgen
```

Comment:
`openssl@1.1` and `util-linux` were indirectly installed via `curl` dependency tree, but are directly needed.

`libressl` is keg-only so was an unnecessary dependency since script needs `openssl` on PATH.

Would have been nicer if our `openssl` alias pointed to the linked copy of OpenSSL so we could use `uses_from_macos "openssl"`, but we current alias `openssl@3` but link `openssl@1.1`.